src/main/java/io/jenkins/plugins/synopsys/security/scan/PluginParametersHandler.java

```diff
@@ -173,7 +173,12 @@ public void logMessagesForParameters(Map<String, Object> scanParameters, Set<Str
             if (key.equals(ApplicationConstants.SYNOPSYS_BRIDGE_DOWNLOAD_URL)
                     || key.equals(ApplicationConstants.SYNOPSYS_BRIDGE_DOWNLOAD_VERSION)
                     || key.equals(ApplicationConstants.SYNOPSYS_BRIDGE_INSTALL_DIRECTORY)
-                    || key.equals(ApplicationConstants.INCLUDE_DIAGNOSTICS_KEY)) {
+                    || key.equals(ApplicationConstants.INCLUDE_DIAGNOSTICS_KEY)
+                    || key.equals(ApplicationConstants.REPORTS_SARIF_CREATE_KEY)
+                    || key.equals(ApplicationConstants.REPORTS_SARIF_FILE_PATH_KEY)
+                    || key.equals(ApplicationConstants.REPORTS_SARIF_ISSUE_TYPES_KEY)
+                    || key.equals(ApplicationConstants.REPORTS_SARIF_SEVERITIES_KEY)
+                    || key.equals(ApplicationConstants.REPORTS_SARIF_GROUPSCAISSUES_KEY)) {
```

Comment:
Can we replace all these SARIF REPORT conditional checks with something like this  - 
`key.startsWith("reports_sarif")` or `key.contains("reports_sarif")` ?

Reply:
I think it is good to be specific rather than just check "reports_sarif" as otherwise it will be error prone for wrong parameter that may start with reports_sarif.

Reply:
We already did this here - https://github.com/jenkinsci/synopsys-security-scan-plugin/pull/35/files#diff-aa19313d6074f10b6438d35835b6f8c4248585a2f30990faf09b5a65f5dbe349L155 for the 3 products so that if new parameters come up for the products then we don't need to modify the code here. But if we don't do this then for new sarif report parameters we need to make changes here every time.